vllm/entrypoints/openai/serving_chat.py

```diff
@@ -195,6 +232,7 @@ async def create_chat_completion(
         if request.stream:
             return self.chat_completion_stream_generator(
                 request, result_generator, request_id, conversation, tokenizer)
+
```

Comment:
It's not at least with hermes and mistral models - I haven't encountered this issue at all. I'm not familiar with that family of models, though I can investigate it if/when I add support for it.

Reply:
maybe it's because the hermes and mistral models's function call tag is not a special token.
i think there need a chance to adjust those request parameters in the framework to support internlm family models which use special token as the function call tag in the generation text.

Reply:
Could be. But, this PR doesn't support Interlm - supporting it would take a separate PR to add a tool parser, which would also be the correct place to investigate making this change.